package.json

```diff
@@ -29,30 +35,33 @@
     "test": "echo \"Error: no test specified\" && exit 0"
   },
   "dependencies": {
-    "postcss-scss": "^4.0.7",
-    "stylelint-order": "^6.0.3",
-    "stylelint-prettier": "^4.0.2",
-    "stylelint-scss": "^5.1.0"
+    "postcss-scss": "^4.0.9",
+    "stylelint-order": "^7.0.0",
+    "stylelint-prettier": "^5.0.3",
+    "stylelint-scss": "^6.11.1"
   },
   "devDependencies": {
-    "@commitlint/cli": "^17.7.1",
-    "@commitlint/config-conventional": "^17.7.0",
+    "@commitlint/cli": "^19.8.1",
+    "@commitlint/config-conventional": "^19.8.1",
     "@gravity-ui/prettier-config": "^1.1.0",
-    "husky": "8.0.3",
+    "husky": "^9.1.7",
     "nano-staged": "^0.8.0",
     "prettier": "^3.0.3",
-    "stylelint": "^15.10.3"
+    "stylelint": "^16.19.1"
   },
   "peerDependencies": {
     "postcss": "^8.0.0",
     "prettier": "^3.0.0",
-    "stylelint": "^15.8.0"
+    "stylelint": "^16.18.0"
```

Comment:
Can we require `^16.0.0` here?

Reply:
UPD: replaced with more fresh deps:` NodeJs >=20` & `stylelint-order 7.0.0` with `stylelint": "^16.18.0` return.